src/resources/information/fr.xhtml

```diff
@@ -188,31 +188,31 @@
     </p>
 
     <ul class="nobullet">
-      <li>Simon Le Roux, concepteur de l'interface utilisateur.</li>
+      <li>Simon Le Roux, UI designer.</li>
       <li>Cyrille Lebeaupin, PDG de noop.fr, développeur senior.</li>
       <li>Jiminy Panoz, créateur de Readium CSS.</li>
-      <li>Léo Stéfani, node.js, React developer.</li>
-      <li>Manuel Lagrand, stagiaire, développeur de node.js.</li>
+      <li>Léo Stéfani, développeur node.js, React.</li>
+      <li>Manuel Lagrand, stagiaire, développeur node.js.</li>
     </ul>
 
-    <p>Le projet est supervisé par Laurent Le Meur, CTO d'EDROab.</p>
+    <p>Le projet est supervisé par Laurent Le Meur, CTO d'EDRLab.</p>
 
     <h2>Qui finance cette application</h2>
 
     <p>
-      L'EDRLab est une organisation à but non lucratif. Notre budget provient
-      essentiellement de nos membres. L'EDRLab a commencé en France, mais compte
-      maintenant 60 membres de Europe, Amérique du Nord, Amérique du Sud et
-      Asie.
+      EDRLab est une organisation à but non lucratif. Notre budget provient
+      essentiellement de nos membres. EDRLab a démarré en France, mais compte
+      maintenant 60 membres d'Europe, d'Amérique du Nord, d'Amérique du Sud et
+      d'Asie.
     </p>
 
-    <p>Ce projet n'aurait pas vu le jour sans:</p>
+    <p>Ce projet n'aurait pas vu le jour sans :</p>
     <p>
       1- L'implication financière de nos membres fondateurs : Editis, Hachette
-      Livre, Magrigall, Media-Participations, Cercle de la Librarie, Syndicat
-      national de l'Edition. Ils comprennent que le financement normes ouvertes
-      et open-source est le meilleur moyen de dynamiser un marché innovant pour
-      l'industrie de l'édition.
+      Livre, Madrigall, Média-Participations, Cercle de la Librarie, Syndicat
+      national de l'Edition. Ils comprennent que le financement de normes 
+      ouvertes et open-source est le meilleur moyen de dynamiser un marché 
```

Comment:
"normes ouvertes" -> "standards ouverts". 
I know, norms vs standards is a subject for long discussion ..

Reply:
fixed in other branch.